ext/standard/var_unserializer.re

```diff
@@ -455,7 +455,7 @@ static zend_always_inline int process_nested_data(UNSERIALIZE_PARAMETER, HashTab
 		if ((*var_hash)->max_depth > 0 && ++(*var_hash)->cur_depth > (*var_hash)->max_depth) {
 			php_error_docref(NULL, E_WARNING,
 				"Maximum depth of " ZEND_LONG_FMT " exceeded. "
-				"You can change the depth limit using the max_depth option",
+				"The depth limit ran be changed by using the max_depth option",
```

Comment:
Typo ``ran`` -> ``can``

Reply:
Also the word "by" isn't necessary. It reads better as "The depth limit can be changed using the max_depth option".

Reply:
Oops, fixed :)